sources/assets/web/css/load.css

```diff
@@ -8,6 +8,7 @@ html, body {
     margin: 0;
     background-color: #36393F;
     height: 100%;
+    -webkit-app-region: drag;
```

Comment:
I saw how it is working, actually. I'm unsure if leaving title-bar hidden is a good idea, as I think it would be great to actually make it functional as close to native title bar as possible, while staying with Discord's look and feel.